src/main/java/org/terasology/core/world/generator/worldGenerators/SimplexFacetedWorldGenerator.java

```diff
@@ -53,7 +52,7 @@ public Vector3f getSpawnPosition(EntityRef entity) {
 
     @Override
     protected WorldBuilder createWorld() {
-        int seaLevel = 32;
+        int seaLevel = 15;
```

Comment:
What is the reason this was changed?

Reply:
In the previous version of the world generator, the sea-level set the overall height scale of the world. Now, it sets the depth of the ocean instead (the deep ocean floor is at 0 height), and this seemed like a reasonable value for that.